doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -2631,18 +2631,19 @@ term for the intermediate lemma, using tactics of the form:
 
 After the :token:`i_pattern`, a list of binders is allowed.
 
+The following example requires the mathcomp and mczify libraries.
```

Comment:
This note should be repeated in the other sections with examples using these libraries, shouldn't it?